docs/overview/set-up-your-dev-environment.md

```diff
@@ -11,7 +11,7 @@ This guide helps you set up tools so you can create Office Add-ins by following
 
 - Node.js
 - npm
-- An Office 365 (the subscription version of Office) account
+- a Microsoft 365 (the subscription version of Office) account
```

Comment:
Uppercase A?

Reply:
Changed to: A Microsoft 365 account which includes the subscription version of Office